doc/user/zebra.rst

```diff
@@ -1334,3 +1334,203 @@ Debugging
 
    Nexthop and nexthop-group events.
 
+Scripting
+=========
+
+Script Commands
+---------------
+
+.. clicmd:: script HOOKCALL SCRIPT
+
+   Set a Lua script for a Lua hook call. HOOKCALL must be a valid Zebra hook
+   call from :ref:`zebra-hook-calls`. SCRIPT is the basename of the script,
+   without `.lua`.
+
+Data structures
+---------------
+
+.. _const-struct-zebra-dplane-ctx:
+
+const struct zebra_dplane_ctx
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
```

Comment:
Not lua, drop the specifier

Reply:
updated in https://github.com/FRRouting/frr/pull/9440/commits/7c4d8a01a50ee0da66295815ce82d5b31357978f